beacon-chain/core/epoch/precompute/reward_penalty_test.go

```diff
@@ -126,54 +126,6 @@ func TestAttestationDeltas_ZeroInclusionDelay(t *testing.T) {
 	require.ErrorContains(t, "attestation with inclusion delay of 0", err)
 }
 
-func TestProcessRewardsAndPenaltiesPrecompute_SlashedInactivePenalty(t *testing.T) {
```

Comment:
NOTE: this is a legacy test for phase0. so deleted whole test, for removing `InactivityPenaltyQuotient`